.gitignore

```diff
@@ -168,3 +168,7 @@ settings.json
 **/*.png
 **/*.pt
 /metrics/
+
+# dev
+temp*.py
+tmp*.py
```

Comment:
Maybe I'll see below, but are python files with these names created by the nnUnet process or somewhere else? Just curious why we need the ignores.

Reply:
No i was just creating some temporary files manually to test things out and put this in there so that I could still use `git add .`. I thought maybe it'd be useful to leave in since anything named temp*.py or tmp*.py in the main folder is likely to be a temporary file. However I can remove it if you think its best to only gitignore whats absolutely necessary